### DISCUSSION TITLE
Feature Request: Storing edit count number

### DISCUSSION THREAD

**Discussion by Fffrankoo** [DynamicPlayerSector/timethings#11] [Ideas]
Would it be possible for your plugin to store edit count information (number of edit sessions) for a note in addition to the "updated_at" and "editing_seconds" information that your plugin already captures? I would add another valuable piece of information for notetakers.

**Comment by plasmabit** [comment]
I was looking for an idea like this! You are right, that would add more depth to the metadata of a note, while being simple and lightweight at the same time. This feature is a must have. I'll get to it as soon as I can.

**Comment by Fffrankoo** [comment]
That's great news! I have been looking for a way to capture this information in the past, but always hit a brick wall. I'm not much of a programmer, so I could not tell how to automatically update this property when you open and modify a note. 
Thank you for spending effort on this! Much appreciated. Best rgds, Frank
